challengeFinal/Coordinator/Coordinator.swift

```diff
@@ -8,9 +8,10 @@
 import Foundation
 import UIKit
 
-protocol Coordinator {
+protocol Coordinator: AnyObject {
```

Comment:
Why are you using AnyObject?

Reply:
The Coordinator must be an Class. Changed to class object.